module_utils/network_lsr/argument_validator.py

```diff
@@ -1182,17 +1183,17 @@ def __init__(self):
         )
 
     def _validate_post(self, value, name, result):
-        if result["key_mgmt"] == "wpa-psk":
+        if result["key_mgmt"] == "wpa-psk" or result["key_mgmt"] == "owe":
```

Comment:
`result["key_mgmt"] in VALID_KEY_MGMT`

Reply:
(key_mgmt: "wpa-eap") does not need the password